blueprints/ember-cli-mirage/index.js

```diff
@@ -32,6 +35,23 @@ module.exports = {
       after: '"predef": [\n'
     });
 
+    if (existsSync('tests/helpers/destroy-app.js')) {
+      this.insertIntoFile('tests/helpers/destroy-app.js', '  server.shutdown();', {
+        after: "Ember.run(application, 'destroy');\n"
+      });
```

Comment:
[Original destroy-app](https://raw.githubusercontent.com/ember-cli/ember-cli/83b9aba4f413ee464f864f9d9f5aab8a88564cf1/blueprints/app/files/tests/helpers/destroy-app.js)

And after this change:
```javascript
import Ember from 'ember';

export default function destroyApp(application) {
  Ember.run(application, 'destroy');
  server.shutdown();
}
```